development-strategy.md

```diff
@@ -106,22 +119,41 @@ Ideas about what the page should have..
 - user can take practice test
 - user can enroll for test
 - user can take exam
+- user can see the results of the test , his correct and incorrect answers.
 - user can write comment/questions/feedback
 
 ## 8. User Story: Admin Page 
 
 * As an admin user can see a page specialized for him.
+* As an admin in the navbar there is an extra link for modifying(add,delete,change..) the sources (tutorials, articles, tests).
+    - As an Expert/Admin/Administrator/Teacher user can see all statistics of the students.
+        - how many registered students are there ? 
+        - how much success do get the students in the tests?
+    - As an Expert/Admin/Administrator/Teacher user can access the sources. 
+        - Expert/Admin/Administrator/Teacher can add/update/delete exam
+        - Expert/Admin/Administrator/Teacher can add/update/delete questions
+        - Expert/Admin/Administrator/Teacher can add/update/delete answers
+        - Expert/Admin/Administrator/Teacher can view test results
+        - Expert/Admin/Administrator/Teacher can grant user to take re-exam
+
+__FRONTEND__
+* there is a `modify sources` button , when he clicks, it goes to the another page.
+    * there should be different folders for sources and tests.
+    * in each folders also there will be related folders. and when clicked and folders he enters the inside of folder and see the content.
+    * in each folder there is an `add` button. 
+    * in the `modified page` there are 2 sections. sources and tests..
+    * buttom of each source there will be `delete`button and `change`button.
+* there is a `statistics` button. when he clicks the button , `render` the results and user sees the statistics. 
+
+__BACKEND__
+
+* when the user `clicked` the `add` button , open a box for uploading , run `add.js` , go to `db` and add the uploaded file to the folder.
+* when a user click a folder then he enters that folder.
+* when `delete` button clicked then first ask that `are you sure to delete this part?`
```

Comment:
I think this can be in frontend part 
* Create `delete` button, when it is clicked show an alert `are you sure to delete this part?`
And backend part:
* When alert answered as `ok` `delete` file from `db`.  and create response as confirmation `The file deleted` or `error`.